Source/OxyPlot/Series/ContourSeries.cs

```diff
@@ -46,8 +47,9 @@ public ContourSeries()
         {
             this.ContourLevelStep = double.NaN;
 
-            this.LabelSpacing = double.NaN;
-            this.LabelStep = 1;
+            this.MultiLabel = false;
+            this.LabelSpacing = 150;
+            this.LabelStep = 0;
```

Comment:
Having `LabelStep == 0` seems to have some dedicated behaviour: could this be explained in the inline documentation?